recipe/meta.yaml

```
@@ -29,34 +29,27 @@ requirements:
     - python
     - setuptools-rust
     - libprotobuf
+    - xz  # [linux64]
   run:
     - python
-    - dask >=2022.3.0,<=2023.5.1
+    - dask >=2022.3.0,<=2023.7.1
     - pandas >=1.4.0
-    # FIXME: handling is needed for httpx-based fastapi>=0.87.0
-    - fastapi >=0.69.0,<0.87.0
+    - fastapi >=0.92.0
```

Am seeing [`httpx` added upstream]( https://github.com/dask-contrib/dask-sql/compare/2023.6.0...2023.8.0#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R49 ). Should we add it here?

```suggestion
    - fastapi >=0.92.0
    - httpx>=0.24.1
```

---

Nice catch! I ended up missing this despite using grayskull to generate a meta.yaml I could look at.

---

Oh interesting did `grayskull` not pick this up? If so, think we should [file a bug report]( https://github.com/conda/grayskull/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%5BBUG%5D+Title+of+the+bug )